test/FunctionalTests/Tests/DataServices/UnitTests/DesignT4UnitTests/ODataT4CodeGeneratorTests.cs

```diff
@@ -113,6 +131,17 @@ public void CodeGenSimpleEdmx()
             ODataT4CodeGeneratorTestDescriptors.Simple.Verify(code, false/*isCSharp*/, true/*useDSC*/);
         }
 
+        [TestMethod]
+        public void CodeGenSimpleEdmxMultipleFiles()
+        {
+            SplitFiles = true;
+            CompileGeneratedCode = false;
+            string code = CodeGenWithT4Template(ODataT4CodeGeneratorTestDescriptors.SimpleMultipleFiles.Metadata, null, true, false);
+            ODataT4CodeGeneratorTestDescriptors.SimpleMultipleFiles.Verify(code, true/*isCSharp*/, false/*useDSC*/);
+            SplitFiles = false;
+            CompileGeneratedCode = true;

```

Comment:
I think it would be better to pass these arguments to the relevant methods instead of setting the class members to some value before the test case and to another after the test case.

Reply:
trues

Reply:
Fixed